app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const { getTopics } = require('./controllers/app.controllers');
+const { handleRouteNotFoundError } = require('./controllers/errors.controllers');
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/topics', getTopics);
+
+// must be after every other route
+app.get('*', handleRouteNotFoundError);
+
+app.use((err, req, res, next) => {
+    if(err.status) {
```

Comment:
Same with this custom error handling - not yet needed for this endpoint so should be removed
(the 500 one's okay to keep though)

Reply:
same